README.md

```diff
@@ -158,16 +158,38 @@ pieces search query --mode fts
 ```
 
 Examines all words in a document to find matches to search criteria
+##### change the llm model you are using:
 
+Lists all the avaiable models on your machine with their indices to choose from .
+
+```bash
+pieces change_model 
+```
+##### change the llm model you are using:
+
+Change the model in the ask command.
+
+```bash
+pieces change_model [MODEL_INDEX]
+```
 ##### Ask a question to a model:
 ** Requires quotes around question **
 
-This currently only supports GPT 3.5 and it does not have working memory. Only coding questions are currently supported. To use the model's code you can copy it from the console and use the create command to create an asset using the copied code. 
+Ask the copoilt a question it uses chatGPT3 as a defualt model to ask a question, you can change the model using the change model command.
 
 ```bash
 pieces ask "your question"
 ```
 
+
+##### Commiting to github
+
+Auto commit the code to github and generate a commit message you can use the `-p` or `--push` flags to push the code to the repo too
```

Comment:
I have done extensive testing for the auto-commit feature, but it doesn't seem to be giving proper results. it gives the same response everytime
<img width="1168" alt="image" src="https://github.com/pieces-app/cli-agent/assets/143828586/7d9d53dc-0bd7-4684-9bbd-e2a69193ace8">


Also we need to perhaps add applications.db in .gitignore

Reply:
Can u please check the .gitignore I should have added that in the last line in gitignore

Reply:
If u have better implementation lemme know or better prompt

Reply:
The main implementation will be in the auto commit file in the api package

Reply:
you should be in the directory where the .git file is in like do cd to change the directory and run it

Reply:
@BishoyHanyRaafat would you mind adding that to this comment, and also ensure that you throw errors in the case there is not a git file in that directory